go.mod

```diff
@@ -7,59 +7,78 @@ require (
 	github.com/google/go-github/v42 v42.0.0
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/sonatype-nexus-community/gonexus v0.59.0
-	github.com/tektoncd/pipeline v0.24.0
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
+	github.com/tektoncd/pipeline v0.28.3
```

Comment:
I forgot to update this in #579. I noticed it in this PR because the `timeouts` field was added between 0.24 and 0.28. The remainder of the changes in `go.mod` and `go.sum` are a result of running `go get github.com/tektoncd/pipeline@v0.28.3 && go mod tidy`.